internal/provider/resource_tsuru_app_autoscale.go

```diff
@@ -62,9 +62,36 @@ func resourceTsuruApplicationAutoscale() *schema.Resource {
 				Required:    true,
 			},
 			"cpu_average": {
-				Type:        schema.TypeString,
-				Description: "CPU average, for example: 20%, mean that we trigger autoscale when the average of CPU Usage of units is 20%.",
-				Required:    true,
+				Type:         schema.TypeString,
+				Description:  "CPU average, for example: 20%, mean that we trigger autoscale when the average of CPU Usage of units is 20%.",
+				Optional:     true,
+				AtLeastOneOf: []string{"cpu_average", "schedules"},
+			},
+			"schedules": {
```

Comment:
@crgisch in terraform, people usually uses lists on singular
https://registry.terraform.io/providers/hashicorp/kubernetes/latest/docs/resources/deployment_v1#container
https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/container_cluster#node_pool

then, let rename "schedules" to schedule and recreate the documentation